cocos/network/WebSocket.cpp

```diff
@@ -101,15 +101,19 @@ class WebSocketCallbackWrapper {
     
     static int onSocketCallback(struct libwebsocket_context *ctx,
                                 struct libwebsocket *wsi,
+#if(CC_TARGET_PLATFORM != CC_PLATFORM_OHOS)
                                 enum libwebsocket_callback_reasons reason,
+#endif
                                 void *user, void *in, size_t len)
     {
         // Gets the user data from context. We know that it's a 'WebSocket' instance.
-        WebSocket* wsInstance = (WebSocket*)libwebsocket_context_user(ctx);
-        if (wsInstance)
-        {
+		#if(CC_TARGET_PLATFORM != CC_PLATFORM_OHOS)
+       		WebSocket* wsInstance = (WebSocket*)libwebsocket_context_user(ctx);
+        	if (wsInstance)
+         {
```

Comment:
Wrong indent.

Reply:
Modified

Reply:
I don't find any change.

Reply:
I have adjusted the indentation and aligned it with the pre line and next lines. Do you want the macro to define without indentation?

Reply:
No. As you can see, after the modification, new codes have different indent with existing codes:
![WeChatf233ec4926dad93cfb30b600a05f8d62](https://github.com/user-attachments/assets/306e9905-c50a-4c7e-888a-0bd52d9d9f2e)

You should see the result in web, not in local editor.

Reply:
We usually use 4 spaces instead of a tab as tab may have different number of spaces in different editor.

Reply:
Modified，Please check it